modules/exploits/linux/http/dlink_dsl2750b_exec_noauth.rb

```diff
@@ -0,0 +1,98 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GreatRanking
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'D-Link DSL-2750B OS Command Injection',
+      'Description'    => %q{
+        This module exploits a remote command injection vulnerability in D-Link DSL-2750B devices.
+        Vulnerability can be exploited through "cli" parameter that is directly used to invoke
+        "ayecli" binary. Vulnerable firmwares are from 1.01 up to 1.03.
+      },
+      'Author'         =>
+        [
+          'p@ql',  # vulnerability discovery
+          'Marcin Bury <marcin[at]threat9.com>'  # metasploit module
+        ],
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          ['PACKETSTORM', 135706],
+          ['URL', 'http://seclists.org/fulldisclosure/2016/Feb/53'],
```

Comment:
CVE? EDB? SecurityFocus? SecurityTracker?

Reply:
I added one more link about this vulnerability. Seems CVE was not assigned and there are no info on EDB, SecurityFocus or SecurityTracker